Calibration/LumiAlCaRecoProducers/python/ALCARECOAlCaPCCRandom_cff.py

```diff
@@ -2,8 +2,8 @@
 
 import HLTrigger.HLTfilters.hltHighLevel_cfi
 ALCARECORandomHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
-    HLTPaths = cms.vstring("AlCa_LumiPixelsCounts_Random_v*"),
-    eventSetupPathsKey='',
+    #HLTPaths = cms.vstring("AlCa_LumiPixelsCounts_Random_v*"),
+    eventSetupPathsKey='AlCa_LumiPixelsCounts_Random_v*',
```

Comment:
I dont think this is the name of the key, is it?

Reply:
```suggestion
    eventSetupPathsKey='AlCaPCCRandom',
```
I think this is what we need and then a new trigger bit in the GT